Spigot-API-Patches/0218-Create-HoverEvent-from-ItemStack-Entity.patch

```diff
@@ -0,0 +1,123 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: ysl3000 <yannicklamprecht@live.de>
+Date: Mon, 6 Jul 2020 22:17:37 +0200
+Subject: [PATCH] Create-HoverEvent-from-ItemStack-Entity
+
+
+diff --git a/pom.xml b/pom.xml
+index 15ce1c66462351d6d1ba55c111db13ef1c9f0789..dc2fb86815d2a3de4b5ff3d22320d234e7ae3436 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -92,7 +92,7 @@
+         <dependency>
+             <groupId>net.md-5</groupId>
+             <artifactId>bungeecord-chat</artifactId>
+-            <version>1.16-R0.1</version>
++            <version>1.16-R0.2-SNAPSHOT</version>
```

Comment:
Should we wait until this is non snapshot and available upstream? Could probably come at that time when 1.16.2 releases?